Copilot/README.md

```diff
@@ -0,0 +1,350 @@
+# SharePoint Embedded copilot
+
+This SDK provides a React component that allows you to add copilot experience to your SharePoint Embedded React application.
+
+_**Please note**:_ This SDK and the custom copilot chat experience is in private preview with limited support. Please consult us at ContactSPECopilot@microsoft.com before deploying this feature to a production environment.
+
+
+## SPE Custom Copilot Overview
+
+This copilot chat control offers the following features:
+- Reason over documents in Sharepoint Embedded containers and user-accessible content in Sharepoint Online.
+- Developer can configure the application code to limit the search scope to file, folder, and containers.
```

Comment:
This line uses `Developer` but the next one uses `Developers`. Stick to `Developers`